tslearn/nonmyopic.py

```diff
@@ -0,0 +1,509 @@
+from sklearn.neural_network import MLPClassifier
+from sklearn.metrics import confusion_matrix
+from sklearn.base import BaseEstimator, ClassifierMixin
+from tslearn.utils import to_time_series_dataset
+from tslearn.clustering import TimeSeriesKMeans, silhouette_score
+import matplotlib.pyplot as plt
+import numpy as np
+import math
```

Comment:
Probably useless, use the corresponding functions from `numpy` instead. Also, please clean the imports once you have removed methods that should not belong here.